<launch>
    <node pkg="al_rtk_ros_driver" type="al_rtk_ros_driver" name="al_rtk_ros_driver">
        <param name="map_frame" value="map"/>
        <param name="serial_device" value="/dev/box_3"/>
        <param name="badurate" value="115200"/>
        <param name="domain" value="rtk.ntrip.qxwz.com"/>
        <param name="account" value="test"/>
        <param name="password" value="test"/>
        <param name="publish_pos_fix_only" value="true" />
        <param name="auto_use_first_fix_as_enu_origin" value="true" />
        <param name="origin_config_file" value="$(find al_rtk_ros_driver)/rtk_tracking_test_enu_origin.json"/>
        <param name="log_dir" value="$(find al_rtk_ros_driver)/log"/>
        <param name="stderrthreshold" value="2"/>
        #log level INFO:0 WARNING:1 ERROR:2 FATAL:3
    </node>
</launch>